### ISSUE TITLE
[BUG] Avatar is not updated

### ISSUE THREAD

**Issue by prsnbrg** [owncloud/android#2602]
### Actual behaviour
-Tell us what happens
I'm updating the avatar in Web UI.
In the app I still see the old avatar despite updating the files etc..

### Expected behaviour
-Tell us what should happen
It would be helpful if the avatars could be updated to keep the servers apart, especially if you have multiple servers.

### Steps to reproduce
1. Update avatar in Web UI
2. Open Android App
3. See old avatar after refresh from the filelist view

Can this problem be reproduced with the official owncloud server?
(url: https://demo.owncloud.org, user: test, password: test)

### Environment data
Android version:
7.1.1
Device model: 
Sony Xperia Z5
Stock or customized system:

ownCloud app version:
2.11.1
ownCloud server version:
10.1.1
#### Web server error log
```
Insert your webserver log here
```

#### ownCloud log (data/owncloud.log)
```
Insert your ownCloud log here
```

**Comment by davigonz** [comment]
Hi @prsnbrg , thanks for reporting this. I will try to reproduce it.

**Comment by jesmrec** [comment]
I managed to reproduce the problem. 

That was a core bug, already fixed in 10.2. The problem was that the core response after asking for the avatar included the display name instead of the pic itself. 

I encourage you to upgrade your server version to the newest (10.2.1), and tell us if you also see the issue fixed.

Thanks for reporting @prsnbrg

**Comment by prsnbrg** [comment]
Yes the error no longer exists with oC 10.2.1.

Great
Thanks @jesmrec
